### PR TITLE
VACMS-8334: Add margin to login page h2.

### LINKED ISSUE #8334: Login page w/o announcement block - spacing wonky
When the announcement block gets removed from the login page, the spacing gets funny. 

ACs
- [ ] Padding/spacing between heading and button match [Figma specs in this frame](https://www.figma.com/file/w0z1NIdY7nent0Q6r7hjTD/Login-page?node-id=304%3A3)

### PR DESCRIPTION
**Pull request by msbtterswrth**
## Description

Closes #8334 .

## Testing done
Disabled the CMS announcement block and logged out. Saw that there was now margin below the heading. 

## Screenshots
image

## QA steps

What needs to be checked to prove this works?  
What needs to be checked to prove it didn't break any related things?  
What variations of circumstances (users, actions, values) need to be checked?  

As CMS admin, 
1. Disable the 'CMS Announcement Block' (`/admin/structure/block`)
2. Then log out
   - [x] Validate that there is now spacing under the title on the login screen
3. Then validate Acceptance Criteria from issue
   - [x] Padding/spacing between heading and button match [Figma specs in this frame](https://www.figma.com/file/w0z1NIdY7nent0Q6r7hjTD/Login-page?node-id=304%3A3)

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Platform CMS Team`
- [ ] `Sitewide CMS Team`
  - [ ] `⭐️ Content ops`
  - [ ] `⭐️ CMS Experience`
  - [ ] `⭐️ Offices`
  - [ ] `⭐️ Product Support`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change.
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: docroot/themes/custom/vagovclaro/assets/scss/pages/_login.scss [hunk 1]
```diff
@@ -36,7 +36,7 @@
   }
 
   h2 {
-    margin: 0;
+    margin: 0 0 36px;
```
[the remark this answers is not in the document]
Comment (msbtterswrth) [reply docroot/themes/custom/vagovclaro/assets/scss/pages/_login.scss]: updated :)

**Comment by msbtterswrth** [comment]
> Also, does the block need to be disabled in config as well, so it doesn't come back on future deploys? I'm unsure

Good question, if Rachel turns it off on prod, will that just come back again next deploy? This is why I typically ignore block placement config haha.

File: docroot/themes/custom/vagovclaro/assets/scss/pages/_login.scss [hunk 2]
```diff
@@ -36,7 +36,7 @@
   }
 
   h2 {
-    margin: 0;
+    margin: var(--spacing-ll);
```
Comment (tjheffner) [inline docroot/themes/custom/vagovclaro/assets/scss/pages/_login.scss:39]: sorry, I think `0 0 var(--spacing-ll)` is correct (you had 0 0 36px before). just want it on the bottom.

**Review by tjheffner — APPROVED** [review]

**Comment by swirtSJW** [comment]
> > Also, does the block need to be disabled in config as well, so it doesn't come back on future deploys? I'm unsure
> 
> Good question, if Rachel turns it off on prod, will that just come back again next deploy? This is why I typically ignore block placement config haha.

@tjheffner 's concern was not really answered here.

**Comment by swirtSJW** [comment]
Discussed on scrum.   This will be merged as is and then a followup issue to either config ignore that one specific block, or remove it in code.

**Pull request merged**